packages/cli/test/commands/fix/__snapshots__/fix.test.ts.snap

```diff
@@ -0,0 +1,312 @@
+[DATA] processing file: /src/apply-rules.ts
+[DATA] processing file: /src/gen-random-grid.ts
+[DATA] processing file: /src/get-live-neighbor-count.ts
+[DATA] processing file: /test/main.test.ts
+[TITLE] Types Inferred
+[TITLE]   16 errors caught by rehearsal
+[TITLE]   6 have been fixed by rehearsal
+[TITLE]   10 errors need to be fixed manually
+[TITLE]     -- 4 ts errors, marked by @ts-expect-error @rehearsal TODO
+[TITLE]     -- 6 eslint errors, with details in the report
+[SUCCESS] Types Inferred
+[SUCCESS]   16 errors caught by rehearsal
+[SUCCESS]   6 have been fixed by rehearsal
+[SUCCESS]   10 errors need to be fixed manually
+[SUCCESS]     -- 4 ts errors, marked by @ts-expect-error @rehearsal TODO
+[SUCCESS]     -- 6 eslint errors, with details in the report"
+`;
+
+exports[`Command: fix "base_ts_app" fixture > fix file with src arg 2`] = `
+"// generate a random grid based on a set dimension and random alive/dead cells
+export function generateRandomGrid(
+  dimensions: number[],
+  chars = ['▄', ' ']
+): string[][]  {
+  const grid = [];
+
+  // create the grid
+  for (let y = 0; y < dimensions[1]; y++) {
+    const row = [];
+    for (let x = 0; x < dimensions[0]; x++) {
+      if (Math.random() > 0.5) {
+        row.push(chars[0]);
+      } else {
+        row.push(chars[1]);
+      }
+    }
+
+    grid.push(row);
+  }
+
+  return grid;
+}
+"
+`;
+
+exports[`Command: fix "base_ts_app" fixture > fix file with src arg 3`] = `""`;
+
+exports[`Command: fix "ember-ts-app" fixture > fix package with src arg and graph 1`] = `
+"info:    @rehearsal/fix<test-version>
+[STARTED] Initialize
+[SUCCESS] Initialize
+[STARTED] Analyzing project dependency graph
+[SUCCESS] Analyzing project dependency graph
+[STARTED] Infer Types
+[DATA] processing file: /app/app.js
+error:   Failed to load parser @typescript-eslint/parser declared in .eslintrc.json: Cannot find module @typescript-eslint/scope-manager
+Require stack:
+- <tmp-path>/node_modules/@typescript-eslint/parser/dist/parser.js
+- <tmp-path>/node_modules/@typescript-eslint/parser/dist/index.js
+- /Users/imac/D/rehearsal-js/node_modules/.pnpm/@eslint+eslintrc@2.0.3/node_modules/@eslint/eslintrc/dist/eslintrc.cjs
+Error: Failed to load parser @typescript-eslint/parser declared in .eslintrc.json: Cannot find module @typescript-eslint/scope-manager
+Require stack:
+- <tmp-path>/node_modules/@typescript-eslint/parser/dist/parser.js
+- <tmp-path>/node_modules/@typescript-eslint/parser/dist/index.js
+- /Users/imac/D/rehearsal-js/node_modules/.pnpm/@eslint+eslintrc@2.0.3/node_modules/@eslint/eslintrc/dist/eslintrc.cjs
+    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:995:15)
+    at Function.Module._load (node:internal/modules/cjs/loader:841:27)
+    at Module.require (node:internal/modules/cjs/loader:1067:19)
+    at require (node:internal/modules/cjs/helpers:103:18)
+    at Object.<anonymous> (<tmp-path>/node_modules/@typescript-eslint/parser/dist/parser.js:7:25)
```

Comment:
Seems incorrect?

Reply:
yes this is known incorrect. i have this test skipped for now while i figure out whats up with the fixture. if you pull out this fixture from the mono-repo it runs fine without this error. looking into it